spanner/client.go

```diff
@@ -1005,7 +1005,12 @@ func (c *Client) prepareTransaction(ctx context.Context, options TransactionOpti
 	t := &ReadWriteTransaction{
 		txReadyOrClosed: make(chan struct{}),
 	}
+	t.txReadOnly.sp = c.idleSessions
 	t.txReadOnly.sh = sh
+	t.txReadOnly.txReadEnv = t
+	t.txReadOnly.qo = c.qo
+	t.txReadOnly.ro = c.ro
+	t.txReadOnly.disableRouteToLeader = c.disableRouteToLeader
```

Comment:
This is added because it's used in begin call 
https://github.com/googleapis/google-cloud-go/blob/spanner/v1.66.0/spanner/transaction.go#L786

qo, ro are not needed but I added it anyways